utils/config/daos.yml

```diff
@@ -31,17 +31,9 @@
 #  allow_insecure: false
 #  # Custom CA Root certificate for generated certs
 #  ca_cert: .daos/daosCA.crt
-#  # Server certificate for use in TLS handshakes
-#  cert: .daos/daos_server.crt
-#  # Key portion of Server Certificate
-#  key: .daos/daos_server.key
-#  # Name of server accodring to its certificate
+#  # Admin certificate for use in TLS handshakes
+#  cert: .daos/admin.crt
+#  # Key portion of Admin Certificate
+#  key: .daos/admin.key
+#  # Name of server according to its certificate
```

Comment:
Can probably remove these two lines from the example config

Reply:
removed